assignments/prototypes.js

```diff
@@ -14,15 +16,31 @@
   * dimensions
   * destroy() // prototype method -> returns the string: 'Object was removed from the game.'
 */
-
+function GameObject (createdAt,dimensions){
+  this.createdAt=createdAt;
+  this.dimensions=dimensions;
+}; 
+GameObject.prototype.destroy = function () {
+  return 'Object was removed from the game'
+}; 
 /*
   === CharacterStats ===
   * hp
   * name
   * takeDamage() // prototype method -> returns the string '<object name> took damage.'
   * should inherit destroy() from GameObject's prototype
 */
+function CharacterStats(hp, name) {
```

Comment:
think we covered this yesterday, remember to keep the same form as the arguments from the original constructor call